src/authentication/serializers.py

```diff
@@ -120,7 +120,7 @@ def create(self, validated_data):
 
         register.send(sender=self.__class__, user=user)
 
-        if not settings.EMAIL_ENABLED:
+        if user.can_login():
```

Comment:
this should be `if not settings.EMAIL_ENABLED and user.can_login()` otherwise we just remove the need for email verification

Reply:
Fixed in 1344b8f51cd08f50991a14595044c10f868733d0